kolibri/core/assets/src/kolibri_app.js

```diff
@@ -110,6 +116,10 @@ export default class KolibriApp extends KolibriModule {
         ...this.stateSetters.map(setter => setter(this.store)),
       ]).then(() => {
         this.startRootVue();
+        window.addEventListener('pointerdown', this.handlePointerDown.bind(this));
```

Comment:
This can be done in the module level of browserInfo.js - it will only get executed once, so it feels safe to me.

In terms of unbinding, once we have detected this once, we don't need to be checking any more, so at that point we could remove the event listener.